### PR TITLE
Organize files into data and docs folders

### PR DESCRIPTION
**Pull request by abluejay-piyo**
Move SQL files and update references in configuration files.

* Move `adminjs.sql` and `adminjs_new.sql` to the `data` folder.
* Update `docker-compose.yaml` to reference `data/adminjs.sql` and `data/scripts/mysql-init.sh`.
* Update `mysql-init.sh` to reference `data/adminjs.sql`.
* Update `README.md` to mention the `data` folder for SQL files and the `docs` folder for documentation files.

---

For more details, open the [Copilot Workspace session](https://copilot-workspace.githubnext.com/mslfh/xt_server/pull/27?shareId=89319b86-ed95-4f50-97cc-71cfee733794).

### CODE REVIEW

**Comment by abluejay-piyo** [comment]
resolved #26

File: docker-compose.yaml
Comment (ErikApoyy) [inline docker-compose.yaml:1]: I think its better to keep this line here as it helps ensure the MySQL readiness:
entrypoint: [ "sh", "/wait-for-mysql.sh" ]

Copilot recommendation:
1. Keeping the line can be beneficial to ensure the application waits for MySQL to be available, which can be useful to handle initial startup scenarios or database readiness.

2. Removing the line might increase the risk of the application failing to connect to the database if MySQL is not ready in time.

Comment (abluejay-piyo) [reply docker-compose.yaml:1]: The script is move to data, not removed. By the way, I google and found:
https://stackoverflow.com/questions/42567475/docker-compose-check-if-mysql-connection-is-ready

Comment (ErikApoyy) [reply docker-compose.yaml:1]: That thread is interesting. It says the 'healthcheck' inside the docker-compose.yaml is already ensuring the mysql readiness. 

When I ask Copilot just now, it also says that's true. Making the script in entrypoint redundant. 

**Copilot response:**
Conclusion
The healthcheck indeed ensures the readiness of the MySQL database before dependent services start, making the wait-for-mysql.sh script redundant.

**Pull request merged**